src/components/avatar-button/styles.ts

```diff
@@ -0,0 +1,23 @@
+import { styled } from '@linaria/react'
+import { ElAvatar } from '../avatar/styles'
+
+export const ElAvatarButton = styled.button`
+  cursor: pointer;
+  background: none;
+  border: none;
+
+  &:focus-visible {
+    outline: none;
+  }
+
+  &:focus-visible ${ElAvatar} {
```

Comment:
as per Design spec, there is a description written under anatomy of `button` component that implies to use `focus-visible` https://www.figma.com/design/pJ96bxoEFStjjIbAPy0Jaj/Design-Guidelines?node-id=945-24543&m=dev

Reply:
> issue: focus should not make the avatar white. Please double check the designs 🙏 It looks like there is no background colour change for focus.

Ah yes, definitely 🤦 

> question: is there a reason for targeting the ElAvatar element here? Why can't we just apply the focus styling to the button itself?

Its possible, but I have to add additional style to the button `border-radius: 100%; padding: 0;`, I just thought to add it to the `el-avatar` is simpler 🤔 , what do you think?

Reply:
> Its possible, but I have to add additional style to the button border-radius: 100%; padding: 0;, I just thought to add it to the el-avatar is simpler 🤔 , I will update that

Ahh, fair point. No leave it as is 👍

Reply:
I just update it and merge without seeing this thread again 🤦 , let me know if I need to revert it in another pr

Reply:
haha no stress. I think your approach was correct. Avatar has two shapes, circle and square, so we want the box-shadow to match the shape of the Avatar, not the button. I believe what you had originally was the simplest way of achieving that 👍 

Not a big deal though.